charts/common/templates/_telemetry.tpl

```diff
@@ -43,7 +43,7 @@
   {{- $serviceProtocol := .serviceProtocol -}}
   {{- $insecure := dig "insecure" true $config -}}
   {{- if eq (dig "endpoint" "auto" $config) "auto" -}}
-    {{- $services := (dict "monitoring" (list "alloy" "open-telemetry-collector-opentelemetry-collector")) -}}
+    {{- $services := (dict "monitoring" (list "telemetry-collector" "alloy" "open-telemetry-collector-opentelemetry-collector")) -}}
```

Comment:
I guess there is a copy paste error. The last element of the list is called `open-telemetry-collector-opentelemetry-collector`. I assume it should be `open-telemetry-collector`

Reply:
Nope, this is the intended name, see https://github.com/teutonet/teutonet-helm-charts/pull/1433